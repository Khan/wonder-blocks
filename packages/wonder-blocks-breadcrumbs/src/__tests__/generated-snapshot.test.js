// This file is auto-generated by gen-snapshot-tests.js
// Do not edit this file.  To make changes to these snapshot tests:
//   1. edit the markdown documentation files in the package,
//        packages/wonder-blocks-breadcrumbs
//   2. Run `yarn run gen-snapshot-tests`.
import React from "react";
import renderer from "react-test-renderer";

// Mock react-dom as jest doesn't like findDOMNode.
jest.mock("react-dom");
import Link from "@khanacademy/wonder-blocks-link";
import {
    Breadcrumbs,
    BreadcrumbsItem,
} from "@khanacademy/wonder-blocks-breadcrumbs";

describe("wonder-blocks-breadcrumbs", () => {
    it("example 1", () => {
        const example = (
            <Breadcrumbs>
                <BreadcrumbsItem>
                    <Link href="https://khanacademy.org/">Home</Link>
                </BreadcrumbsItem>
                <BreadcrumbsItem>
                    <Link href="https://khanacademy.org/about">About</Link>
                </BreadcrumbsItem>
                <BreadcrumbsItem>Current page</BreadcrumbsItem>
            </Breadcrumbs>
        );
        const tree = renderer.create(example).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
