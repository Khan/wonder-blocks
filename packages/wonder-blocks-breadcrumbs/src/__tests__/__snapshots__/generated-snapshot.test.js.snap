// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wonder-blocks-breadcrumbs example 1 1`] = `
<nav
  aria-label="Breadcrumbs"
>
  <ol
    className=""
    style={
      {
        "display": "flex",
        "listStyle": "none",
        "margin": 0,
        "overflow": "hidden",
        "padding": 0,
      }
    }
  >
    <li
      className=""
      style={
        {
          "alignItems": "center",
          "display": "flex",
          "justifyContent": "center",
          "marginRight": 4,
        }
      }
    >
      <a
        className=""
        href="https://khanacademy.org/"
        onBlur={[Function]}
        onClick={[Function]}
        onDragStart={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchCancel={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
        style={
          {
            "color": "#1865f2",
            "cursor": "pointer",
            "outline": "none",
            "textDecoration": "none",
          }
        }
        tabIndex={0}
      >
        Home
      </a>
      <svg
        aria-hidden={true}
        className=""
        height={16}
        style={
          {
            "marginLeft": 4,
          }
        }
        viewBox="0 0 16 16"
        width={16}
      >
        <circle
          cx="8"
          cy="9"
          r="1.5"
        />
      </svg>
    </li>
    <li
      className=""
      style={
        {
          "alignItems": "center",
          "display": "flex",
          "justifyContent": "center",
          "marginRight": 4,
        }
      }
    >
      <a
        className=""
        href="https://khanacademy.org/about"
        onBlur={[Function]}
        onClick={[Function]}
        onDragStart={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchCancel={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
        style={
          {
            "color": "#1865f2",
            "cursor": "pointer",
            "outline": "none",
            "textDecoration": "none",
          }
        }
        tabIndex={0}
      >
        About
      </a>
      <svg
        aria-hidden={true}
        className=""
        height={16}
        style={
          {
            "marginLeft": 4,
          }
        }
        viewBox="0 0 16 16"
        width={16}
      >
        <circle
          cx="8"
          cy="9"
          r="1.5"
        />
      </svg>
    </li>
    <li
      aria-current="page"
      className=""
      style={
        {
          "alignItems": "center",
          "display": "flex",
          "justifyContent": "center",
          "marginRight": 4,
        }
      }
    >
      Current page
    </li>
  </ol>
</nav>
`;
